lib/iris/tests/test_pandas.py

```diff
@@ -29,9 +29,13 @@
 import cf_units
 import matplotlib.units
 import netCDF4
-import netcdftime
 import numpy as np
 
+# TODO Remove this if statement as we move to cftime
+if cf_units.__version__ > '1':
```

Comment:
Is this necessary? Do you expect the tests to be passing with netcdftime still?

Reply:
> Is this necessary? Do you expect the tests to be passing with netcdftime still?

Probably not. Let me remove that...